mlnode/iotdb/mlnode/parser.py

```diff
@@ -0,0 +1,125 @@
+import argparse
+from iotdb.mlnode.exception import ModelNotSupportedError, MissingConfigError
+
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+
+class ConfigParser(object):
+    def __init__(self):
+        self.parser = argparse.ArgumentParser()
+        self.argument_list = []
+        self.required_list = []
+
+    def parse_configs(self, config):
+        conf_dict = vars(self.parser.parse_args([]))
+        args = []
+        for k in self.required_list:
+            if k not in config.keys():
+                raise MissingConfigError(k)
```

Comment:
Would use `required=True` in `parse.add_argument()` directly better?
For example:
`self.parser.add_argument('--source_type', type=str, required=True)`

Reply:
As you can see in method `parse_args`， I use 'conf_dict = vars(self.parser.parse_args([]))' to get conf_dict in order to get each argument's type, which can be used for type check. If i set some argument's `required` attribute to be True, I cannot use `parse_args([])`. So I should check this manually.